voice/bxml/callbacks/recordComplete.md

```diff
@@ -17,24 +17,27 @@ Content-Type: application/xml; charset=utf-8
+| callId            | The call id associated with the event.                                                                                                                                                                |
+| parentCallId      | (optional) If the event is related to the B leg of a `<Transfer>`, this is set to the call id of the original call leg that contained the `<Transfer>` tag. Otherwise, null.                          |
+| recordingId       | The unique id for this recording.                                                                                                                                                                     |
+| callUrl           | The URL of the call associated with the event.                                                                                                                                                        |
+| mediaUrl          | URL to retrieve the contents of the recording.                                                                                                                                                        |
+| startTime         | Time the recording was started, in ISO 8601 format.                                                                                                                                                   |
+| endTime           | Time the recording ended, in ISO 8601 format.                                                                                                                                                         |
+| duration          | Duration of the recording, in ISO 8601 format.                                                                                                                                                        |
+| channels          | Number of channels in the recording.                                                                                                                                                                  |
+| fileFormat        | The audio format that the recording was saved as (`wav` or `mp3`).                                                                                                                                    |
+| tag               | (optional) The `tag` specified earlier in the call. If no `tag` was specified or it was previously cleared, null.                                                                                     |
+| transferCallerId  | (optional) If the event is related to the B leg of a `<Transfer>`, this is set to the `from` field of the B-leg call, in E.164 format (e.g. +15555555555). Otherwise, null.                           |
+| transferTo        | (optional) If the event is related to the B leg of a `<Transfer>`, this is set to the phone number used as the `to` field of the B-leg call, in E.164 format (e.g. +15555555555). Otherwise, null.    |
```

Comment:
Whatever suggestions you decided to incorporate from `gather`'s callbacks, make sure to do the same here and in all the other relevant callbacks.